src/features/moduleCalls.ts

```diff
@@ -10,6 +17,18 @@ export class ModuleCallsFeature implements StaticFeature {
 
   constructor(private client: BaseLanguageClient, private view: ModuleCallsDataProvider) {}
 
+  fillInitializeParams?: ((params: InitializeParams) => void) | undefined;
+
+  preInitialize?:
+    | ((capabilities: ServerCapabilities<any>, documentSelector: vscode.DocumentSelector | undefined) => void)
+    | undefined;
+
```

Comment:
```suggestion
```

Since these two methods are optional, we don't have to implement them. Only `getState` is required. The same applies to the other static features below as well.